Engine/include/food.h

```diff
@@ -5,32 +5,35 @@
 
 #include "environment.h"
 
-class Food : public Entity {
+class Food : public Entity {  
  protected:
   Food(const double nutritional_value);
   Food(const double x_coord, const double y_coord, const double nutritional_value);
   Food(const double x_coord, const double y_coord, const double size, const double nutritional_value);
   Food(const double size, const double nutritional_value);
 
  public:
+  enum type { meat, plant };
   void Eat();
   void SetNutritionalValue(double value);
   double GetNutritionalValue() const;
 
   void SetLifespan(int lifespan);
   int GetLifespan() const;
+  type GetType() const;
 
  protected:
+  Food::type type_;
```

Comment:
the nutritional values per unit are used from the constants. So, I think we don't really need the nutritional value as another variable.